#! /bin/sh /usr/share/dpatch/dpatch-run
## 32_debian_nolsb.dpatch
## by Petter Reinholdtsen <pere@hungry.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Make sure to not install the LSB init.d include file and
## DP: functions.  These are in other packages on debian.

@DPATCH@
diff -urNad insserv~/Makefile insserv/Makefile
--- insserv~/Makefile	2008-01-04 11:22:01.000000000 +0100
+++ insserv/Makefile	2008-01-04 11:22:01.000000000 +0100
@@ -75,15 +75,15 @@
 	$(MKDIR)   $(SBINDIR)
 	$(MKDIR)   $(SDOCDIR)
 	$(MKDIR)   $(CONFDIR)
-	$(MKDIR)   $(LSBDIR)
-	$(MKDIR)   $(DESTDIR)/usr/lib
-	$(MKDIR)   $(USRLSBDIR)
+#	$(MKDIR)   $(LSBDIR)
+#	$(MKDIR)   $(DESTDIR)/usr/lib
+#	$(MKDIR)   $(USRLSBDIR)
 	$(INSTBIN) insserv        $(SBINDIR)/
 	$(INSTDOC) insserv.8      $(SDOCDIR)/
 	$(INSTCON) insserv.conf   $(CONFDIR)/
-	$(INSTCON) init-functions $(LSBDIR)/
-	$(INSTSRP) install_initd  $(USRLSBDIR)/
-	$(INSTSRP) remove_initd   $(USRLSBDIR)/
+#	$(INSTCON) init-functions $(LSBDIR)/
+#	$(INSTSRP) install_initd  $(USRLSBDIR)/
+#	$(INSTSRP) remove_initd   $(USRLSBDIR)/
 
 #
 # Make distribution
